docs/GenericsManifesto.md

```diff
@@ -110,15 +110,15 @@ extension Collection {
 
 `let` constants could be allowed to have generic parameters, such that they produce differently-typed values depending on how they are used. For example, this is particularly useful for named literal values, e.g.,
 
-```let π<T : ExpressibleByFloatLiteral>: T = 3.141592653589793238462643383279502884197169399```
+```Swift let π<T : ExpressibleByFloatLiteral>: T = 3.141592653589793238462643383279502884197169399```
```

Comment:
Please add a line break after "Swift" and after "399", otherwise the word "Swift" is just displayed in a monospace font.